### PR TITLE
Add plugin Customer Email Report to the Store

### PR DESCRIPTION
**Pull request by simonschaufi**

### CODE REVIEW

**Review by kevinpapst — CHANGES_REQUESTED** [review]
please add german translation to: 
https://github.com/kimai/www.kimai.org/blob/main/_data/de/store.yml

File: _data/developer.yml [hunk 1]
```diff
@@ -161,3 +161,22 @@ helsinkisystems:
...
+    username: simonschaufi
+    paypal: https://www.paypal.me/simonschaufi
+    github_donate: https://github.com/sponsors/simonschaufi
+    purchase_details:
+        - title: Where can I buy this plugin?
+          content: You can buy the plugin via Lemon Squeezy (CreditCard, instant access)
+        - title: Where can I get support for this plugin?
+          content: Support-questions, bug-reports and feature-requests can be made via the contact form at <a href="https://www.simonschaufelberger.de/de/kontakt.html" target="_blank">www.simonschaufelberger.de</a>.
+        - title: Can I get access to the GitHub Repository of this plugin?
+          content: Yes, after purchasing the plugin, I can give you access to the GitHub Repository of this plugin if you tell me your GitHub username via the contact form at <a href="https://www.simonschaufelberger.de/de/kontakt.html" target="_blank">www.simonschaufelberger.de</a>.
```
[the remark this answers is not in the document]
Comment (simonschaufi) [reply _data/developer.yml:182]: Why did you remove so much? Is this not relevant?

[the remark this answers is not in the document]
Comment (kevinpapst) [reply _data/developer.yml:182]: Most of the keys are not displayed anywhere

File: _data/store/simonschaufi-customer-email-report-bundle.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+developer: simonschaufi
+icon: fas fa-home
+demo: false
+subscription: 9&nbsp;€
```
[the remark this answers is not in the document]
Comment (kevinpapst) [reply _data/store/simonschaufi-customer-email-report-bundle.yml:4]: Fixe

**Comment by simonschaufi** [comment]
> please add german translation to: https://github.com/kimai/www.kimai.org/blob/main/_data/de/store.yml

done

**Comment by kevinpapst** [comment]
The translations are normally done by the community. 
I just want to have english and german for the start, because those are the biggest markets.

**Comment by simonschaufi** [comment]
> The translations are normally done by the community. I just want to have english and german for the start, because those are the biggest markets.

I used DeepL for my translations which are available.

File: _data/store/simonschaufi-customer-email-report-bundle.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+developer: simonschaufi
+icon: fas fa-home
```
Comment (kevinpapst) [inline _data/store/simonschaufi-customer-email-report-bundle.yml:2]:
```suggestion
icon: fas fa-envelope
```

Comment (simonschaufi) [reply _data/store/simonschaufi-customer-email-report-bundle.yml]: Where is this icon used? And why did you remove the editorconfig?

**Comment by simonschaufi** [comment]
@kevinpapst Why did you remove the editorconfig? This file is extremely helpful for code style.

Where is this icon used?

**Comment by kevinpapst** [comment]
Maybe it isn't currently, I don't know. It was on the plugin listing page in the past. If I ever activate it again, it should be something useful.

Editorconfig can be a new PR. But it works without as well.
This is YAML and HTML and Liquid, so it is either clear or doesn't matter. 
I am to old to waste my lifetime with whitespace discussions.

**Comment by simonschaufi** [comment]
ok. thank you for your answer.

**Pull request merged**